docs/components/saas/byok/faq.md

```diff
@@ -0,0 +1,97 @@
+---
+id: faq
+title: "FAQ"
+description: "Frequently asked questions about encryption at rest, encryption key types, and External Encryption Keys in Camunda 8 SaaS."
+keywords: ["encryption", "BYOK", "KMS", "encryption keys", "security", "FAQ"]
+---
+
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+
+This page answers common questions about encryption at rest, encryption key types, and External Encryption Keys for Camunda 8 SaaS clusters.
+
+## General questions
+
+### What is encryption at rest?
+
+Encryption at rest ensures that data stored on physical media (for example, disks or backups) is encrypted, protecting it from unauthorized access if storage is compromised.
```

Comment:
Can we add qualifiers when it applies to the Orcheastrion cluster and when to Web Modeller?

Reply:
I _think_ my added note resolves your comment here, but let me know if this is still unclear.